### PR TITLE
{math,numlib}[foss/2020a,intel/2020a] gmsh v4.7.1, SLEPc v3.12.2 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by besserox**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/gmsh/gmsh-4.7.1-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+separate_build_dir = True
+
+configopts = '-DENABLE_FLTK=0 -DENABLE_WRAP_PYTHON=ON -DENABLE_METIS=1 '
+configopts += '-DENABLE_BUILD_LIB=ON -DENABLE_BUILD_SHARED=ON -DENABLE_BUILD_DYNAMIC=ON'
+
+modextrapaths = {'PYTHONPATH': ['lib64']}
+
+sanity_check_paths = {
+    'files': ['bin/gmsh', 'lib64/gmsh.py'],
```
Comment (boegel) [inline easybuild/easyconfigs/g/gmsh/gmsh-4.7.1-foss-2020a-Python-3.8.2.eb:36]: @besserox Should we enhance this a bit, now that the shared library is also built?

```suggestion
    'files': ['bin/gmsh', 'lib64/gmsh.py', 'lib64/libgmsh.a', 'lib64/libgmsh.%s' % SHLIB_EXT],
```

**Review by boegel — DISMISSED** [review]
lgtm

File: easybuild/easyconfigs/g/gmsh/gmsh-4.7.1-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (besserox) [reply easybuild/easyconfigs/g/gmsh/gmsh-4.7.1-foss-2020a-Python-3.8.2.eb]: pushed the correction for foss and intel

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @besserox !

**Pull request merged**